lib/wanda/catalog.ex

```diff
@@ -9,13 +9,19 @@ defmodule Wanda.Catalog do
     Fact
   }
 
-  @catalog_path Application.compile_env!(:wanda, [__MODULE__, :catalog_path])
+  @doc """
+  Get checks catalog path
+  """
+  def get_catalog_path() do
+    local_path = Application.fetch_env!(:wanda, Wanda.Catalog)[:catalog_path]
+    Application.app_dir(:wanda, local_path)
```

Comment:
Why are we now getting the full application path?

Reply:
Read PR description please XD

Reply:
Sorry, it wasn't immediately obvious to me. I'm guessing it has to do with the path changing for a realease build thus requiring an absolute path? Nevermind, fine by me if it works :ok_hand: